server/events/working_dir.go

```diff
@@ -35,7 +35,7 @@ const workingDirPrefix = "repos"
 type WorkingDir interface {
 	// Clone git clones headRepo, checks out the branch and then returns the
 	// absolute path to the root of the cloned repo.
```

Comment:
can you update these docs for what the bool return value indicates

Reply:
Done